docs/source/markdown/podman-create.1.md

```diff
@@ -1412,6 +1412,18 @@ $ podman start --attach ctr
 b
 ```
 
+## CONMON
+
+When Podman starts a container it actually executes the conmon program, which
+then executes the OCI Runtime.  Conmon is the container monitor.  It is a small
+program who’s job is to watch the primary process of the container, and if the
+container dies, save the exit code.  It also holds open the tty of the
+container, so that it can be attached to later. This is what allows Podman to
+run in detached mode (backgrounded), so Podman can exit but conmon continues to
+run.  Each container has their own instance of conmon. When Podman interacts
+with the container (stop, exec, restart ...), it is actually communicating with
```

Comment:
Not true - we're communicating with the OCI runtime, which doesn't interact with Conmon at all. Conmon is really just sitting, waiting for an exit, while we deal directly with container PID1

Reply:
Give me better description then.

Reply:
Check it out now.

Reply:
Works for me